xml/System.IO/File.xml

```diff
@@ -1189,11 +1189,11 @@
         <Parameter Name="fileSecurity" Type="System.Security.AccessControl.FileSecurity" Index="3" FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="path">The name of the file.</param>
+        <param name="path">The path and name of the file to create.</param>
         <param name="bufferSize">The number of bytes buffered for reads and writes to the file.</param>
         <param name="options">One of the <see cref="T:System.IO.FileOptions" /> values that describes how to create or overwrite the file.</param>
         <param name="fileSecurity">One of the <see cref="T:System.Security.AccessControl.FileSecurity" /> values that determines the access control and audit security for the file.</param>
-        <summary>Creates or overwrites the specified file with the specified buffer size, file options, and file security.</summary>
+        <summary>Creates or overwrites a file in the specified path, specifying a buffer size, a <see cref="T:System.IO.FileOptions" /> value that describes how to create or overwrite the file and a <see cref="T:System.Security.AccessControl.FileSecurity" /> value that determines the access control and audit security for the file.</summary>
```

Comment:
```suggestion
        <summary>Creates or overwrites a file in the specified path, specifying a buffer size, options that describe how to create or overwrite the file, and a value that determines the access control and audit security for the file.</summary>
```

Reply:
Hello @rpetrusha, I agree with your suggestions, but I noticed a small discrepancy between them:
In this suggestion, you added a comma before "and", but in the other suggestion you didn't add a comma.
Should both have a comma before "and"?

Reply:
No. Only the second should have a comma, @JeroenOortwijn.

Reply:
Ah, ok!